code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -234,7 +234,8 @@
 
 /datum/reagent/medicine/rezadone
 	name = "Rezadone"
-	description = "A powder derived from fish toxin, Rezadone can effectively treat genetic damage as well as restoring minor wounds and restoring corpses husked by burns. Overdose will cause intense nausea and minor toxin damage."
+	//description = "A powder derived from fish toxin, Rezadone can effectively treat genetic damage as well as restoring minor wounds and restoring corpses husked by burns. Overdose will cause intense nausea and minor toxin damage."
+description = "A powder derived from fish toxin, Rezadone can effectively treat genetic damage as well as restoring minor wounds and restoring corpses husked by burns or dessication. Overdose will cause intense nausea and minor toxin damage." //SKYRAT EDIT CHANGE
```

Comment:
```suggestion
  description = "A powder derived from fish toxin, Rezadone can effectively treat genetic damage as well as restoring minor wounds and restoring corpses husked by burns or dessication. Overdose will cause intense nausea and minor toxin damage." //SKYRAT EDIT CHANGE
```
You forgot indentation.

Reply:
Tfw your phone code suggestions are committed